tests/installation/bootloader_zkvm.pm

```diff
@@ -76,8 +77,15 @@ sub run() {
         my $basename = basename($hdd);
         chomp(my $hdd_path = `find $hdd_dir -name $basename | head -n1`);
         diag("HDD path found: $hdd_path");
-        type_string("# copying image...\n");
-        $svirt->add_disk({file => $hdd_path, dev_id => 'a'});    # Copy disk to local storage
+        if (get_var('PATCHED_SYSTEM')) {
+            diag('in patched systems just load the patched image');
+            my $patched_img = "$img_path/$name" . "a.img";
+            $svirt->add_disk({file => $patched_img, dev_id => 'a'});
+        }
+        else {
+            type_string("# copying image...\n");
+            $svirt->add_disk({file => $hdd_path, backingfile => 1, dev_id => 'a'});    # Copy disk to local storage
```

Comment:
I guess we definitely need the 'PATCHED_SYSTEM' branch. Do you argue that we don't need the else branch? but it has been there in before so you are saying it should also be changed additionally or is there a patch where this patch needs to be rebased on?